profiles/package.mask

```diff
@@ -17,6 +17,37 @@ dev-haskell/accelerate-io
 app-text/lit
 dev-haskell/highlighting-kate
 
+# Masked for removal. These require <dev-haskell/lens-4.19, but we're bumping
+# lens up to 4.19. All other upstreams have updated their dependency list to
+# support the new lens, but these have not. These also appear to be abandoned
+# projects.
+dev-haskell/reflex-basic-host
+dev-haskell/reflex-basic-socket
+dev-haskell/reflex-basic-wai
```

Comment:
CI is failing. Please check that you have masked all reverse dependencies of these packages.

Reply:
Yup I've started from scratch in a new branch, off the latest master, systematically going through the reverse deps now.

Third time's the charm :)